financial-services-accelerator/react-apps/self-care-portal/src/main/java/org/wso2/financial/services/accelerator/scp/webapp/util/Utils.java

```diff
@@ -163,27 +169,33 @@ public static LocalDateTime parseEncodedStringToDate(String dateStr) {
     @Generated(message = "Ignoring since method do not contain a logic")
     public static void returnResponse(HttpServletResponse resp, int statusCode, JSONObject payload) {
         try {
-            LOG.debug("Returning response to frontend. payload: " + payload.toString());
+            LOG.debug(String.format("Returning response to frontend. payload: %s",
```

Comment:
Shall we add a check here , isDebugEnabled?

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/752/commits/1e2edab95303b5a528064f8a4ceb3acb3d75fd36